go/conversation/templates/conversation/bulk_send.html

```diff
@@ -0,0 +1,102 @@
+{% extends "app.html" %}
+{% load crispy_forms_tags %}
+
+{% block content_title %}{{ action.action_display_name }}{% endblock %}
+
+{% block content_actions_right %}
+<div class="pull-right">
+    <button name="schedule" class="btn btn-primary" value="Schedule" data-toggle="modal" data-target="#schedule-modal">{{ action.action_schedule_verb|default:"Schedule" }}</button>
```

Comment:
Looking at this, I wonder if we could put this modal into the base action template and just wrap it with `{% if action.action_schedule_verb %}`? It would make it easier to add basic scheduling to other actions (e.g. dialogue push) but harder to customize scheduling if needed and we'd need to document that forms that set `action_schedule_verb` need to have a `scheduled_datetime`.

Thoughts?